{"uid":"42dfcd600255b0a2","name":"test_003","fullName":"cases.test_homepage.TestCase001#test_003","historyId":"88e815a6ef9cf2f452ce394d0df20b83","time":{"start":1680256018641,"stop":1680256018641,"duration":0},"status":"broken","statusMessage":"AttributeError: module 'allure' has no attribute 'environment'","statusTrace":"@pytest.fixture(scope='session', autouse=True)\n    def environment_info():\n        # 收集环境信息\n        env_info = {\n            '浏览器': 'Chrome',\n            '测试网站': 'http://47.107.116.139/phpwind/',\n            '测试环境': 'staging',\n        }\n    \n        # 在Allure报告中添加环境信息\n        with allure.step('添加环境信息'):\n            for key, value in env_info.items():\n>               allure.environment(key, value)\nE               AttributeError: module 'allure' has no attribute 'environment'\n\nconftest.py:56: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":2,"retriesStatusChange":true,"beforeStages":[{"name":"environment_info","time":{"start":1680256018564,"stop":1680256018565,"duration":1},"status":"broken","statusMessage":"AttributeError: module 'allure' has no attribute 'environment'\n","statusTrace":"  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/luna/Documents/learning_python/testing/ui_phpwind/conftest.py\", line 56, in environment_info\n    allure.environment(key, value)\n","steps":[{"name":"添加环境信息","time":{"start":1680256018564,"stop":1680256018564,"duration":0},"status":"broken","statusMessage":"AttributeError: module 'allure' has no attribute 'environment'\n","statusTrace":"  File \"/Users/luna/Documents/learning_python/testing/ui_phpwind/conftest.py\", line 56, in environment_info\n    allure.environment(key, value)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"story","value":"首页测试2"},{"name":"feature","value":"首页测试"},{"name":"parentSuite","value":"cases"},{"name":"suite","value":"test_homepage"},{"name":"subSuite","value":"TestCase001"},{"name":"host","value":"lunadeMacBook-Pro.local"},{"name":"thread","value":"2981-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"cases.test_homepage"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ac3c5b7efce79f3","status":"failed","statusDetails":"AssertionError: assert 1 == 2","time":{"start":1680253720839,"stop":1680253720839,"duration":0}},{"uid":"288051f5ab093d04","status":"failed","statusDetails":"AssertionError: assert 1 == 2","time":{"start":1680250487727,"stop":1680250487728,"duration":1}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"42dfcd600255b0a2.json","parameterValues":[]}